src/types.ts

```diff
@@ -4,7 +4,7 @@ export const AUTH_CHAIN_HEADER_PREFIX = 'x-identity-auth-chain-'
 export const AUTH_TIMESTAMP_HEADER = 'x-identity-timestamp'
 export const AUTH_METADATA_HEADER = 'x-identity-metadata'
 
-export const DEFAULT_CATALYST = 'https://peer-lb.decentraland.org'
+export const DEFAULT_PROVIDER_URL = 'https://rpc.decentraland.org'
```

Comment:
I am concerned that at some point we have to down the endpoint or limit it in some way and everything start failing because we assumed that is granted.

At least, I would like to identify all the use from this library using the `project` param (same that we use on [`decentraland-connect`](https://github.com/decentraland/decentraland-connect/blob/master/src/configuration.ts#L67))